networking_oneview/ml2/drivers/oneview/neutron_oneview_client.py

```diff
@@ -58,13 +58,17 @@ def _is_physnet_in_uplinkset_mapping(self, physical_network, network_type):
         return False
 
     def check_server_hardware_availability(self, server_hardware):
-        while True:
+        max_number_of_attempts = common.CONF.oneview.retries_to_lock_sh
+
+        for i in range(max_number_of_attempts):
             if not server_hardware.get('powerLock'):
                 return True
             time.sleep(30)
```

Comment:
need to create a parameter for this too.

Reply:
Maybe `retries_to_lock_sh_interval`